environment.yml

```
@@ -13,4 +13,4 @@ dependencies:
   - nbconvert
   - ipykernel  # needed to run jupyter notebooks
   - pip:
-    - festim==1.3
+    - festim==1.3.1
```

```suggestion
    - festim==1.3
```

No need to specify the patch version here. If you specify `1.3` then this is equivalent to `1.3.*` which means that if we release another patch version, this will be automatically updated here.

This is ok since patch versions will only contain bug fixes and no breaking changes.